### DISCUSSION TITLE
Release v1.3.0.0

### DISCUSSION THREAD

**Discussion by jlanz** [dreamworksanimation/openmoonray#113] [Announcements]
### Build Improvements
* Initial Rocky 9 support
* Fixed missing #includes
* Use avx2-i32x8 for ISPC
* Enable building with GCC 11
* Enable building with Qt 5.15.3
* Allow Python executable name to be set
* Allow alternate names for Boost::python
* Upgraded to Embree 3.13.5
* Upgraded to ISPC 1.20.0
* Upgraded to CUDA 12.1
* Upgraded to microhttpd 0.9.71
### Features
* Improvements to rdl2_print
* Multi-machine rendering: Added debug utility functions and debugConsole commands
### Bug Fixes
* Fixed vector/scalar look difference related to energy compensation
* Fixed inconsistencies in scalar / vector normal mapping
* Fixed NaNs and crash caused by bad texture data
* Fixed cryptoUV data corruption on resume from checkpoint
* Fixed dropped cryptomatte weights on resume from checkpoint
* Fixed incorrect resume history information related to signal-based checkpoint
* Fixed OpenVdbMap input interface
* Fixed frame locked noise on volumes
* Fixed NaN in bad texture read
* Multi-machine rendering: Fixed arras/moonray vector mode display image refresh issue
### Other
* Improved SceneVariables attribute descriptions
* Improved LightSet and ShadowSet attribute descriptions
* Improved several Material attribute descriptions
* Improved signal-based checkpoint log message

<hr /><em>This discussion was created from the release <a href='https://github.com/dreamworksanimation/openmoonray/releases/tag/openmoonray-1.3.0.0'>Release v1.3.0.0</a>.</em>

**Comment by threedslider** [comment]
Nice work ! And good job to all team :)
